hotsos/core/ycheck/common.py

```diff
@@ -261,9 +262,10 @@ def validate(self, plugin_name):
         search belongs.
         """
         if self.plugin_name is None:
-            raise Exception(f"{self.__class__.__name__}.plugin_name must set "
-                            "to the name of the plugin in which it will run "
-                            f"(currently={self.plugin_name})")
+            raise NameNotSetError(
+                f"{self.__class__.__name__}.plugin_name must set "
```

Comment:
nit: "must be set" - if you change the other typo

Reply:
done